activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -354,17 +354,24 @@ def group!(*args) # :nodoc:
       self
     end
 
-    # Allows to specify an order attribute:
+    # Order can be used to sort the results of a query.
     #
+    # #order accepts arguments in one of several formats.
```

Comment:
```suggestion
    # Order accepts arguments in one of several formats.
```

Reply:
This is designed to look like `where`'s documentation, and displays as a method.

https://github.com/rails/rails/blob/dfef44716a6a0fc50c44e12218e65e51e6ee88bf/activerecord/lib/active_record/relation/query_methods.rb#L560-L562

![image](https://user-images.githubusercontent.com/8124558/129624319-fb7a9c86-e381-497d-8ef2-5ab2a29ccfd9.png)

Reply:
Ah sorry, my mistake 😄